src/User/User.php

```diff
@@ -3,16 +3,18 @@
 namespace Yiisoft\Yii\Web\User;
 
 use Psr\EventDispatcher\EventDispatcherInterface;
+use Psr\Http\Message\ResponseInterface;
 use Yiisoft\Access\AccessCheckerInterface;
 use Yiisoft\Auth\IdentityInterface;
 use Yiisoft\Auth\IdentityRepositoryInterface;
+use Yiisoft\Yii\Web\Cookie;
 use Yiisoft\Yii\Web\Session\SessionInterface;
 use Yiisoft\Yii\Web\User\Event\AfterLogin;
 use Yiisoft\Yii\Web\User\Event\AfterLogout;
 use Yiisoft\Yii\Web\User\Event\BeforeLogin;
 use Yiisoft\Yii\Web\User\Event\BeforeLogout;
 
-class User
+class User implements IdentityInterface
```

Comment:
[That's not correct](https://github.com/yiisoft/yii-web/pull/235#discussion_r409841382). That's not an identity.

Reply:
But here https://github.com/yiisoft/yii-demo/blob/master/src/Entity/User.php#L29 use the IdentityInterface. Not is clear for me. Thanks!

Reply:
Yes. The interface implementation should be done in an entity and should _not_ be done in the service in this pull request. The service should use the entity.

Reply:
Ok perfect i will remove the interface!

Reply:
I updated the PR with this!